test/mailers/user_mailer_test.rb

```diff
@@ -1,7 +1,7 @@
 require 'test_helper'
 
 class UserMailerTest < ActionMailer::TestCase
-  test "the truth" do
+  test 'the truth' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.